service/controller/v14/resource/service/master_service.go

```diff
@@ -20,6 +20,10 @@ func newMasterService(customObject v1alpha1.KVMConfig) *apiv1.Service {
 				"cluster":  key.ClusterID(customObject),
 				"customer": key.ClusterCustomer(customObject),
 				"app":      key.MasterID,
+				"giantswarm.io/cluster":        key.ClusterID(customObject),
+				"giantswarm.io/etcd_domain":    key.ClusterEtcdDomain(customObject),
+				"giantswarm.io/organization":   key.ClusterCustomer(customObject),
+				"giantswarm.io/version_bundle": key.VersionBundleVersion(customObject),
```

Comment:
aligning with aws operator

Reply:
Can we please have constants in the key package and use these here? IMO the workers should have these labels as well.

Reply:
Why do you think that worker should have the labels?

Reply:
I thought our labels should be on all resources we manage.